index.js

```diff
@@ -142,13 +142,20 @@ function SendStream (req, path, options) {
     this._dotfiles = undefined
   }
 
-  this._extensions = opts.extensions !== undefined
-    ? normalizeList(opts.extensions, 'extensions option')
```

Comment:
`normalizeList()` now return an empty list when the input is `undefined`, so the check here is bloated.